rs/bitcoin/kyt/src/providers.rs

```diff
@@ -119,15 +131,16 @@ fn btcscan_request(txid: Txid, max_response_bytes: u32) -> CanisterHttpRequestAr
     }
 }
 
-fn make_request(
+fn make_get_request(
     host: &str,
-    network: BtcNetwork,
+    network: &BtcNetwork,
     txid: Txid,
     max_response_bytes: u32,
 ) -> CanisterHttpRequestArgument {
     let url = match network {
         BtcNetwork::Mainnet => format!("https://{}/api/tx/{}/raw", host, txid),
         BtcNetwork::Testnet => format!("https://{}/testnet/api/tx/{}/raw", host, txid),
+        BtcNetwork::Regtest { .. } => panic!("Request to regtest network requires POST"),
```

Comment:
It seems very weird to me that regtest doesn't offer a raw api like the other providers

Reply:
Note that `getrawtransaction` distinguishes between transactions in blocks and transactions in the mempool, so usage is not straightforward.

Reply:
Yes, essentially what @THLO said. It was not straightforward to use `getrawtransaction`.

Reply:
ha good point, I should have read the description more carefully
> By default this function only works for mempool transactions.